package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@storybook/addon-queryparams",
-  "version": "7.0.1",
+  "version": "8.0.0",
```

Comment:
The Storybook team recommends using v9 to denote compatibility with core. Might be a good idea to do it here too: https://github.com/nuxt-modules/storybook/issues/907

Reply:
Correct, this should be 9.0.0 as there isn't a semantic-release job in place to update based on commits.

@hacknug FYI, wen moving this package to community, we'd want to add a commit that copies tooling from https://github.com/storybook-community/storybook-source-link that automates releases.